### PR TITLE
Add missing dependencies for task plugin

### PR DESCRIPTION
**Pull request by evidolob**
Signed-off-by: Yevhen Vydolob <yvydolob@redhat.com>

### CODE REVIEW

File: plugins/task-plugin/package.json [hunk 1]
```diff
@@ -58,5 +58,9 @@
   },
   "theiaPlugin": {
     "backend": "lib/task-plugin-backend.js"
+  },
+  "dependencies": {
+    "inversify": "^4.14.0",
```
Comment (benoitf) [inline plugins/task-plugin/package.json]: remove the caret ? as CQ should be linked

**Comment by RomanNikitenko** [comment]
thank you, Yevhen!

**Comment by evidolob** [comment]
@RomanNikitenko You need to create CQ for `"inversify": "4.14.0"` and `"reflect-metadata": "0.1.10"`

**Comment by benoitf** [comment]
FYI you may use reflect-metadata 0.1.8 https://dev.eclipse.org/ipzilla/show_bug.cgi?id=12001 already approved and piggy back https://dev.eclipse.org/ipzilla/show_bug.cgi?id=17257 (inversify 4.13.0) to speed up the process

**Comment by RomanNikitenko** [comment]
I am going to test my changes for "inversify": "4.14.0" and "reflect-metadata": "0.1.10" and create CQ

**Comment by RomanNikitenko** [comment]
To speed up the process I tested it for "inversify": "4.13.0" and "reflect-metadata": "0.1.8" - it works fine for me.

**Comment by RomanNikitenko** [comment]
The Piggyback for  "inversify": "4.13.0" https://dev.eclipse.org/ipzilla/show_bug.cgi?id=19424

**Pull request merged**